Drivers/Threshold_workout.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+import sys
+import time
+import os
+from mqtt_client import MQTTClient
+from FTP_class import FTP
```

Comment:
We want to remove this FTP class, as we are using the threshold class for this workout, correct?